components/smartmontools/smartmontools.p5m

```diff
@@ -21,37 +21,42 @@
 # Copyright (c) 2014 Nexenta Systems, Inc.  All rights reserved.
 #
 
-set name=pkg.fmri value=pkg:/storage/smartmontools@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
-set name=pkg.summary value="smartmontools contains utilities that control and monitor storage devices"
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
 set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
-set name=info.classification value="org.opensolaris.category.2008:Applications/System Utilities"
-set name=info.upstream-url value=http://www.smartmontools.org
 set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
 
-license COPYING license='GPLv2'
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
 
 dir path=var/lib/smartmontools
 
 file smartmontools.xml path=lib/svc/manifest/system/smartmontools.xml
 
 file path=etc/smartd.conf preserve=true
+file path=etc/smartd_warning.sh
```

Comment:
Is this shell script really supposed to be in /etc? Also if it is to be executed you need to append mode=0555